packages/core/src/types/config/index.ts

```diff
@@ -139,18 +139,32 @@ type PrismaLogDefinition = {
   emit: 'stdout' | 'event';
 };
 
+type PrismaDatasource = {
+  url?: string;
+};
+
+type PrismaDatasources = {
+  sqlite?: PrismaDatasource;
+};
+
+type PrismaClientConfig = {
+  datasources?: PrismaDatasources;
+  log?: (PrismaLogLevel | PrismaLogDefinition)[];
+};
+
 export type DatabaseConfig<TypeInfo extends BaseKeystoneTypeInfo> = {
   provider: DatabaseProvider;
   url: string;
 
   shadowDatabaseUrl?: string;
   onConnect?: (args: KeystoneContext<TypeInfo>) => Promise<void>;
-  enableLogging?: boolean | PrismaLogLevel | Array<PrismaLogLevel | PrismaLogDefinition>;
+  enableLogging?: boolean | Array<PrismaLogLevel | PrismaLogDefinition>;
   idField?: IdFieldConfig;
   prismaClientPath?: string;
   prismaSchemaPath?: string;
 
   extendPrismaSchema?: (schema: string) => string;
+  prismaClient?: (config: PrismaClientConfig) => any;
```

Comment:
I know that the core doesn't pull in the `@prisma/client` for various reasons, I don't know if its desired to have this type here.

Reply:
The problem is that some code bases might have many different `@prisma/client`'s, so you can't rely on that type.